Library/ViewModels/CategorySelectionViewModel.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public protocol CategorySelectionViewModelInputs {
+  func viewDidLoad()
+}
+
+public protocol CategorySelectionViewModelOutputs {
+  // A tuple of Section Titles: [String], and Categories Section Data: [[(String, PillCellStyle)]]
+  var loadCategorySections: Signal<([String], [[(String, PillCellStyle)]]), Never> { get }
+}
+
+public protocol CategorySelectionViewModelType {
+  var inputs: CategorySelectionViewModelInputs { get }
+  var outputs: CategorySelectionViewModelOutputs { get }
+}
+
+public final class CategorySelectionViewModel: CategorySelectionViewModelType,
+  CategorySelectionViewModelInputs, CategorySelectionViewModelOutputs {
+  public init() {
+    let categoriesEvent = self.viewDidLoadProperty.signal
```

Comment:
I noticed that the screen stays blank until the data is loaded into the CollectionView. It may be a good idea to add an ActivityIndicator to give some visual feedback to users.

Reply:
Yep totally, I think out of the scope of this PR though.

Reply:
Maybe a quick way to achieve that could be to confirm the continue button to `LoadingButton` 🤔  Or prefix the sectionTitles with a title called 'Loading...' ? 😄 But then we're need translations.